maxim/models/test_run.py

```
@@ -385,23 +396,35 @@ def from_json(cls, json_str: str) -> "TestRunEntry":
     @classmethod
     def dict_to_class(cls, data: Dict[str, Any]) -> "TestRunEntry":
         local_evaluation_results = data.get("localEvaluationResults")
+        data_entry = data.get("dataEntry", {})
+        variables = {}
+        for key, value in data_entry.items():
+            if value is not None:
+                variables[key] = Variable.from_json(value)
+
         return cls(
             output=data.get("output", None),
```

Ruff (0.11.9)

Use `data.get("output")` instead of `data.get("output", None)`

Replace `data.get("output", None)` with `data.get("output")`

(SIM910)